src/main/java/org/nebula/contrib/ngbatis/config/EnvConfig.java

```diff
@@ -20,6 +22,8 @@
 @Configuration
 public class EnvConfig {
 
+    public static boolean reconnect = true;

```

Comment:
is there any code to reset `reconnect`?

Reply:
Not. I want to leave it to the developers to make a decision in their project.

Reply:
should we provide a setReconnect function?